cmd/test/test.go

```diff
@@ -0,0 +1,228 @@
+
+The test command expects one or more input files that will be evaluated
+against Open Policy Agent policies. Directories are also supported as valid
+inputs.
+
+Policies are written in the Rego language. For more
+information on how to write Rego policies, see the documentation:
+https://www.openpolicyagent.org/docs/latest/policy-language/
+
+The policy location defaults to the policy directory in the local folder.
+The location can be overridden with the '--policy' flag, e.g.:
+
+	$ ec test --policy <my-directory> <input-file(s)/input-folder>
+
+Some policies are dependant on external data. This data is loaded in separately
+from policies. The location of any data directory or file can be specified with
+the '--data' flag. If a directory is specified, it will be recursively searched for
+any data files. Right now any '.json' or '.yaml' file will be loaded in
+and made available in the Rego policies. Data will be made available in Rego based on
+the file path where the data was found. For example, if data is stored
+under 'policy/exceptions/my_data.yaml', and we execute the following command:
+
+	$ ec test --data policy <input-file>
+
+The data is available under 'import data.exceptions'.
+
+The test command supports the '--output' flag to specify the type, e.g.:
+
+	$ ec test -o table -p examples/kubernetes/policy examples/kubernetes/deployment.yaml
+
+Which will return the following output:
++---------+----------------------------------+--------------------------------+
+| RESULT  |               FILE               |            MESSAGE             |
++---------+----------------------------------+--------------------------------+
+| success | examples/kubernetes/service.yaml |                                |
+| warning | examples/kubernetes/service.yaml | Found service hello-kubernetes |
+|         |                                  | but services are not allowed   |
++---------+----------------------------------+--------------------------------+
+
+By default, it will use the regular stdout output. For a full list of available output types, see the of the '--output' flag.
+
+The test command supports the '--update' flag to fetch the latest version of the policy at the given url.
+It expects one or more urls to fetch the latest policies from, e.g.:
+
+	$ ec test --update opa.azurecr.io/test
+
+See the pull command for more details on supported protocols for fetching policies.
+
+When debugging policies it can be useful to use a more verbose policy evaluation output. By using the '--trace' flag
+the output will include a detailed trace of how the policy was evaluated, e.g.
+
+	$ ec test --trace <input-file>
+`
+
+// NewTestCommand creates a new test command.
+func NewTestCommand() *cobra.Command {
+	cmd := cobra.Command{
+		Use:   "test <path> [path [...]]",
+		Short: "Test your configuration files using Open Policy Agent",
+		Long:  testDesc,
+		PreRunE: func(cmd *cobra.Command, args []string) error {
```

Comment:
If the intention is to check that each of these flags have a value or to make them required, see https://pkg.go.dev/github.com/spf13/cobra#MarkFlagRequired and https://github.com/spf13/cobra/blob/v1.7.0/shell_completions.go#L38 for better way of achieving this.

Reply:
As mentioned, the idea is to make it the same as conftest. I want to avoid refactoring conftest's flag handling if I can.